src/main/java/stsjorbsmod/actions/SowAction.java

```diff
@@ -0,0 +1,25 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import stsjorbsmod.patches.ActionShouldPersistPostCombatField;
+
+public class SowAction extends AbstractGameAction {
+    public SowAction(int sowDamage) {
+        addToBot(new DamageAllEnemiesAction(AbstractDungeon.player, DamageInfo.createDamageMatrix(sowDamage, true),
```

Comment:
Why is this in the constructor instead of `update()`?

Reply:
Good question, moving.